pages/habit-lists/[id].js

```diff
@@ -46,7 +47,37 @@ const List = () => {
     }
     fetchHabits();
   }, [])
-  
+
+  const messageGenerator = () => {
+    const messages = ["Pick the easiest thing first!", "You’re making great progress—keep going!", "you’re on a roll!", "Crushing it!", "You completed all your habits!"];
```

Comment:
Having these messages in an array doesn't really do much --in fact, it makes it harder for a person reading your code to understand what's going on. I think we can just have the message strings in the switch statement below.